lib/elixir/lib/kernel/special_forms.ex

```diff
@@ -1753,6 +1753,22 @@ defmodule Kernel.SpecialForms do
   catch values also support `:error`, as in Erlang, although it is
   commonly avoided in favor of raise/rescue control mechanisms.
 
+  ## After clauses
+
+  An `after` clause works like an `ensure` or `finally` clause in other languages.
```

Comment:
I agree with @whatyouhide. I didn't feel comfortable when I read it, because if in order to explain something we need the user to know other languages, we are not doing a good job at explaining it.